src/screens/TransactionConfirmationScreen.js

```diff
@@ -486,45 +664,169 @@ const TransactionConfirmationScreen = () => {
-          backgroundColor="transparent"
-          dappName={dappName || ''}
-          imageUrl={imageUrl || ''}
-          size={60}
-          style={{ marginBottom: 24 }}
-        />
-        <Text
-          align="center"
-          color="white"
-          letterSpacing="roundedMedium"
-          size="h4"
-          weight="semibold"
-        >
-          {dappName}
-        </Text>
-        <TransactionType>{requestHeader}</TransactionType>
-        <CancelButtonContainer>
-          <Button
-            backgroundColor={colors.alpha(colors.grey, 0.4)}
-            onPress={onCancel}
-            showShadow={false}
-            size="small"
-            textProps={{ color: colors.black, size: 'lmedium' }}
+    <AnimatedContainer
+      style={isReanimatedAvailable ? animatedContainerStyles : fallbackStyles}
+    >
+      <SlackSheet
+        backgroundColor={colors.transparent}
+        borderRadius={0}
+        height={sheetHeight}
+        hideHandle
+        scrollEnabled={false}
+      >
+        <Column>
+          <AnimatedSheet
+            backgroundColor={colors.white}
+            borderRadius={39}
+            direction="column"
+            paddingBottom={isMessageRequest ? safeAreaInsetValues.bottom : 0}
+            paddingHorizontal={19}
+            paddingTop={24}
+            style={animatedSheetStyles}
           >
-            {lang.t('wallet.action.reject')}
-          </Button>
-        </CancelButtonContainer>
-      </Masthead>
-      {renderTransactionSection()}
-    </Container>
+            <SheetHandleFixedToTop showBlur={false} />
+            <Column marginBottom={17} />
+            <DappLogo
+              dappName={dappName || ''}
+              imageUrl={overrideLogo || imageUrl || ''}
+            />
+            <Row marginBottom={5}>
+              <Text
+                align="center"
+                color={colors.alpha(colors.blueGreyDark, 0.8)}
+                letterSpacing="roundedMedium"
+                size="large"
+                weight="bold"
+              >
+                {authenticatedName || formattedDappUrl}
+                {//We only show the checkmark
+                // if it's on the override list (dappNameHandler.js)
+                authenticatedName && (
+                  <Text
+                    align="center"
+                    color={colors.appleBlue}
+                    letterSpacing="roundedMedium"
+                    size="large"
+                    weight="bold"
+                  >
+                    {' 􀇻'}
+                  </Text>
+                )}
+              </Text>
+            </Row>
+            <Centered marginBottom={24} paddingHorizontal={24}>
+              <Text
+                align="center"
+                color={methodName ? 'dark' : 'white'}
+                letterSpacing="roundedMedium"
+                size="larger"
+                weight="heavy"
+              >
+                {methodName || 'Placeholder'}
```

Comment:
We want to keep the "Placeholder" string as the fallback text?

Reply:
It's not visible. It has the same background color until we have a real value. It's a hack to prevent the height of the Sheet changing when we get the method name.